sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/PartitionKind.java

```diff
@@ -10,7 +10,13 @@
     /**
```

Comment:
This needs to be annotated with @Beta as well.

Reply:
Done